### ISSUE TITLE
DUDA TEST 017__Hybrid_Laws_of_Sacae (E4_1 TESTS)

### ISSUE THREAD

**Issue by MTaisa** [franciscogazitua/IIC2113-2024#104]
Holaa, tengo una pequeña duda sobre el siguiente test en el que se prueba la habilidad "Laws of Sacae":

Round 2: Shez (Player 2) comienza
Ninguna unidad tiene ventaja con respecto a la otra
Shez obtiene Atk+6
Shez obtiene Spd+6
Shez obtiene Def+6
Shez obtiene Res+6
Marth no podrá contraatacar
Shez ataca a Marth con 22 de daño
Shez ataca a Marth con 22 de daño
Shez (51) : Marth (10)

Como se puede ver se activa el efecto que anula el contraataque que se activa "si el rival usa espada, lanza o hacha y la Spd de la unidad ≥ Spd del rival +5". Aunque la primera se cumpliría en el test, no se porque se considera que la segunda condición de las Spd es cierta, ya que el Spd de Shez es de 57 y el de Marth es 53. 

No hay otras habilidades que puedan afectar y probando haciendo que se tome en cuenta los bonus +6 anteriores ocurre que en otros tests se aplica el efecto cuando no debería ocurrir por diferencias de spd.

**Comment by RodrigoToroIcarte** [comment]
Efectivamente. Lo que ocurre es que la condición "_Spd de la unidad ≥ Spd del rival +5_" considera bonus, penalties y neutralizadores (de lo contrario, la habilidad diría que se utiliza la Spd base de la unidad).

Entonces creo que el problema está en los "otros tests" que te están fallando una vez que consideras el bonus de +6. ¿Quizás ese bonus es neutralizado por otra habilidad?
